### DISCUSSION TITLE
Welcome to micromdm Discussions!

### DISCUSSION THREAD

**Discussion by groob** [micromdm/micromdm#718] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by meta-github** [comment]
Hi,

my name is Thomas. I am working for a company that is using macOS systems mainly for developers. The number of systems is about 100. We are using a fully centralized administration for these systems and to enable the switch to Big Sur, we just have started to using micromdm. I do have a strong background in unix (I started with SunOS in 1991, later Linux in 1992/1993 and I did maintain a very large open source software package named teTeX for about 10 years). I have never used google go before, so I fear that I will not be able to contribute code in the near future. I can, however share ideas and my experiences that I make.

Thats it for the moment!

Thomas
